scripts/install_rstudio.sh

```diff
@@ -128,9 +129,9 @@ cp /rocker_scripts/pam-helper.sh /usr/lib/rstudio-server/bin/pam-helper
 # Clean up
 rm -rf /var/lib/apt/lists/*
 
-# Check the RStudio Server version
+# Check the RStudio Server
+echo -e "Check the RStudio Server installation...\n"
+rserver --verify-installation 1
```

Comment:
I am not familiar with the rserver command and am wondering if this is the correct way to do it.
This was used on the following page.
https://cran.r-project.org/bin/linux/fedora/

Reply:
Since the image size appears to have increased (1743MB -> 1786MB) with the addition of this, it is removed once at beb72a47ffbd6bc5faac45c17288b488ce4fc312.